.github/workflows/release.yaml

```
@@ -0,0 +1,90 @@
+        with:
+          fetch-depth: 0
+
+      - name: Checkout repo
+        if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/'))
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Push images
+        if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/'))
+        uses: ./.github/actions/publish-and-sign
```

actionlint (1.7.4)

missing input "PROJECT_NAME" which is required by action "Publish and Sign Snapshot Image" defined at "./.github/actions/publish-and-sign". all required inputs are "GITHUB_TOKEN", "IMAGE_TAGS", "PROJECT_NAME", "REGISTRY_ADDRESS", "REGISTRY_PASSWORD", "REGISTRY_USERNAME"

(action)